### ISSUE TITLE
travis-ci is broken

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg-import#30]
https://travis-ci.org/borgbackup/borg-import/pull_requests

job #33 worked, since #35 it is crashing.

**Comment by Ankit-rana** [comment]
It looks like we might not be using right version of virtualenv. 
virtualenv integrated --no-download option in [v14.0.0](https://virtualenv.pypa.io/en/latest/changes/#v14-0-0-2016-01-19)
`+fakeroot -u tox -r

/home/travis/.venv/lib/python3.4/site-packages/packaging/requirements.py:57: UserWarning: warn_ungrouped_named_tokens_in_collection: setting results name 'specifier' on And expression collides with '_original_end' on contained expression

  VERSION_SPEC = originalTextFor(_VERSION_SPEC)("specifier")

/home/travis/.venv/lib/python3.4/site-packages/packaging/requirements.py:60: UserWarning: warn_ungrouped_named_tokens_in_collection: setting results name 'marker' on And expression collides with '_original_end' on contained expression

  MARKER_EXPR = originalTextFor(MARKER_EXPR())("marker")

GLOB sdist-make: /home/travis/build/borgbackup/borg-import/setup.py

py34 create: /home/travis/build/borgbackup/borg-import/.tox/py34

ERROR: invocation failed (exit code 2), logfile: /home/travis/build/borgbackup/borg-import/.tox/py34/log/py34-0.log

================================== log start ===================================

Usage: virtualenv.py [OPTIONS] DEST_DIR

virtualenv.py: error: no such option: --no-download

=================================== log end ====================================

ERROR: InvocationError for command /home/travis/.venv/bin/python -m virtualenv --no-download --python /home/travis/.venv/bin/python py34 (exited with code 2)

___________________________________ summary ____________________________________

ERROR:   py34: InvocationError for command /home/travis/.venv/bin/python -m virtualenv --no-download --python /home/travis/.venv/bin/python py34 (exited with code 2)

The command "./.travis/run.sh" exited with 1.`

**Comment by Ankit-rana** [comment]
I have not been able to find why Tox creates the command having virtualenv with --no-download option

Ran the script with upgraded virtualenv
```
(.venv)travis@08a98a486117:~/borgbackup/borg-import$ pip install --upgrade virtualenv
Collecting virtualenv
  Downloading https://files.pythonhosted.org/packages/f7/69/1ad2d17560c4fc60170056dcd0a568b83f3453a2ac91155af746bcdb9a07/virtualenv-16.7.4-py2.py3-none-any.whl (3.3MB)
    100% |████████████████████████████████| 3.4MB 227kB/s 
Installing collected packages: virtualenv
  Found existing installation: virtualenv 13.1.2
    Uninstalling virtualenv-13.1.2:
      Successfully uninstalled virtualenv-13.1.2
Successfully installed virtualenv-16.7.4
You are using pip version 7.1.2, however version 19.2.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
(.venv)travis@08a98a486117:~/borgbackup/borg-import$ tox -e py34
GLOB sdist-make: /home/travis/borgbackup/borg-import/setup.py
py34 create: /home/travis/borgbackup/borg-import/.tox/py34
py34 installdeps: -rrequirements.d/development.txt
py34 inst: /home/travis/borgbackup/borg-import/.tox/.tmp/package/1/borg-import-0.1.dev47+gf7ebe87.zip
py34 installed: DEPRECATION: Python 3.4 support has been deprecated. pip 19.1 will be the last one supporting it. Please upgrade your Python as Python 3.4 won't be maintained after March 2019 (cf PEP 429).,atomicwrites==1.3.0,attrs==19.1.0,borg-import==0.1.dev47+gf7ebe87,coverage==4.5.4,filelock==3.0.12,importlib-metadata==0.19,more-itertools==7.2.0,packaging==19.1,pathlib2==2.3.4,pluggy==0.12.0,py==1.8.0,pyparsing==2.4.2,pytest==4.6.5,pytest-cov==2.7.1,scandir==1.10.0,six==1.12.0,toml==0.10.0,tox==3.13.2,virtualenv==13.1.2,wcwidth==0.1.7,zipp==0.5.2
py34 run-test-pre: PYTHONHASHSEED='2215208388'
py34 run-test: commands[0] | py.test -rs --cov=borg_import --cov-config=../.coveragerc --pyargs borg_import.helpers.testsuite
======================================================= test session starts =======================================================
platform linux -- Python 3.4.6, pytest-4.6.5, py-1.8.0, pluggy-0.12.0
cachedir: .tox/py34/.pytest_cache
rootdir: /home/travis/borgbackup/borg-import, inifile: setup.cfg
plugins: cov-2.7.1
collected 5 items                                                                                                                 

py34/lib/python3.4/site-packages/borg_import/helpers/testsuite/test_discover.py ..                                          [ 40%]
py34/lib/python3.4/site-packages/borg_import/helpers/testsuite/test_names.py .                                              [ 60%]
py34/lib/python3.4/site-packages/borg_import/helpers/testsuite/test_timestamps.py ..                                        [100%]

----------- coverage: platform linux, python 3.4.6-final-0 -----------
Name                                                                 Stmts   Miss Branch BrPart  Cover
------------------------------------------------------------------------------------------------------
py34/lib/python3.4/site-packages/borg_import/__init__.py                 0      0      0      0   100%
py34/lib/python3.4/site-packages/borg_import/__main__.py                 2      2      0      0     0%
py34/lib/python3.4/site-packages/borg_import/_version.py                 1      1      0      0     0%
py34/lib/python3.4/site-packages/borg_import/helpers/__init__.py         0      0      0      0   100%
py34/lib/python3.4/site-packages/borg_import/helpers/discover.py        18      0     12      1    97%
py34/lib/python3.4/site-packages/borg_import/helpers/names.py           14      0      6      0   100%
py34/lib/python3.4/site-packages/borg_import/helpers/timestamps.py      16      3      2      0    83%
py34/lib/python3.4/site-packages/borg_import/main.py                   147    147     32      0     0%
py34/lib/python3.4/site-packages/borg_import/rsnapshots.py              12     12      4      0     0%
py34/lib/python3.4/site-packages/borg_import/rsynchl.py                 12     12      4      0     0%
------------------------------------------------------------------------------------------------------
TOTAL                                                                  222    177     60      1    23%


==================================================== 5 passed in 0.16 seconds =====================================================
_____________________________________________________________ summary _____________________________________________________________
  py34: commands succeeded
  congratulations :)
```

**Comment by Ankit-rana** [comment]
After a thorough evaluation of failed and successful builds, found out that tox and pytest have been upgraded. if i control it's version in requirement.txt, i get a successful build.
I will be raising a patch with controlled tox and pytest version.

**Comment by ThomasWaldmann** [comment]
Fixed by #32.
